best-practices/client-side/framework/react/components/media-cropper/src/components/media-cropper/renderers/image.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+
+export default class ImageRenderer extends Component {
+    ctx = null;
```

Comment:
So we are trying to enforce more strict typing in our JS projects at Shift3 with either [Flow](https://flow.org/en/docs/react/) or [Typescript](https://www.typescriptlang.org/docs/handbook/react.html). I would recommend we add that to this project before we merge it into the reusable components.